dotnet/src/Extensions/Planning.StepwisePlanner/StepwisePlanner.cs

```diff
@@ -518,7 +513,7 @@ protected internal virtual SystemStep ParseResult(string input)
         return result;
     }
 
-    private async Task<string> InvokeActionAsync(string actionName, Dictionary<string, string> actionVariables)
+    private async Task<string> InvokeActionAsync(string actionName, Dictionary<string, string> actionVariables, CancellationToken cancellationToken)
     {
         var availableFunctions = await this.GetAvailableFunctionsAsync().ConfigureAwait(false);
```

Comment:
The cancellation token can be passed to the `GetAvailableFunctionsAsync()` method.